gems/pending/openstack/openstack_handle/handled_list.rb

```diff
@@ -0,0 +1,118 @@
+module OpenstackHandle
+  module HandledList
+    def handled_list(collection_type, options = {})
```

Comment:
@blomquisg right so handled was meant to be:

handled == handle pagination and multitenancy (and coming soon handle multi domains and who knows future brings) If people want to know details, they can looks at the method which documents it. As a black box, should be fine to know it's just handled :-D

so yeah, I was picking something generic that says, everything is handled for you, the list contains all entities of this collection type. 

So maybe rather :list_all, :list_everything, :list_all_of, ... not sure what is best. handled_list sounds best to me :-)